test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -1276,18 +1313,17 @@ if Version.match?(System.version(), ">= 1.13.0") do
       assert_formatter_output(
         """
         <p>
-          text text <a class="text-blue-500" href="" target="_blank" attr1="">link</a>text.
+          first first <a class="text-blue-500" href="" target="_blank" attr1="">link</a>second.
         </p>
         """,
         """
         <p>
-          text text
-          <a
+          first first <a
```

Comment:
This may seem wrong but the inline element `a` is followed by a text `second` without whitespace causing it to sticky on the same line and breaking its attrs into new lines because of the lack of space in the same line caused by  `line_length: 50`